t/Common.t

```diff
@@ -50,6 +50,16 @@ ok($sis->excludeLane(2), "Excluded lane 2");
 ok($sis->excludeLane(3), "Excluded lane 3");
 ok($sis->excludeLane(3), "Excluded lane 3");
 
+# Test if MD5 sum can be calculated
+my $md5 = $sis->getMd5("$testfolder1/sisyphus.yml", -noCache=>1);
+ok($md5 eq "ffe051d77d496c756e817f90ed5e905e", "MD5 sum was calculated");
+
+# Test if MD5 sum can be written to file
+$sis->saveMd5("$testfolder1/sisyphus.yml", $md5);
```

Comment:
To be honest, I was assuming NFSLock works on all filesystems. If it doesn't, there needs to be code that handles all cases. Do you know or can you check if it is exclusive to NFS mounts @MatildaAslin?